docs/install/post-install.rst

```diff
@@ -302,9 +355,8 @@ The configuration item to use ZFS should look like:
    Some of the suggested steps are not straightforward.
    FLOC-2092
 
-The pool name must match a ZFS storage pool that you have created on all of the Flocker agent nodes.
-This requires first installing :ref:`ZFS on Linux <installing-ZFS-CentOS-7>`.
-You must also set up SSH keys at ``/etc/flocker/id_rsa_flocker`` which will allow each Flocker dataset agent node to authenticate to all other Flocker dataset agent nodes as root.
+The pool name must match a ZFS storage pool that you have created on all of the Flocker agent nodes (see the installing :ref:`ZFS on Linux <installing-ZFS-CentOS-7>` steps above).
```

Comment:
As a reader, my expectation was that this would link to the ZFS on Linux website http://zfsonlinux.org/
I'd suggest changing it to that - I'm not sure what benefit there is to linking back to a couple of paragraphs above on the same page, especially when it's within the same section (http://build.clusterhq.com/results/docs/install-renamewhatnext-FLOC-2724/build-9030/install/post-install.html#zfs-peer-to-peer-backend-configuration-experimental)

Another alternative is to link the ZoL website from the top of that section where we say "The ZFS backend uses node-local storage and ZFS filesystems as the storage for datasets."